api/nodes/serializers.py

```diff
@@ -25,6 +25,33 @@ def to_internal_value(self, data):
         return data
 
 
+class JSONAPINodeListSerializer(JSONAPIListSerializer):
+    """
+    Bulk updates instance with the validated data.
+
+    Request either completely succeeds or fails. Requires
+    the request to be in the serializer context.
+    """
+
+    # Overrides JSONAPIListSerialize which doesn't support multiple update by default.
+    def update(self, instance, validated_data):
+        data_mapping = {item.get('_id', None): item for item in validated_data}
+        request = self.context['request']
+        auth = Auth(request.user)
+        ret = []
+        for node_id, data in data_mapping.items():
+            node = get_object_or_error(Node, node_id, 'node')
+            if node.can_edit(auth) is False:
```

Comment:
I'm still wondering if this necessary. Shouldn't the permissions classes handle this?